test/jdk/javax/swing/UIDefaults/6622002/bug6622002.java

```diff
@@ -21,16 +21,16 @@
  * questions.
  */
 
-/**
+import javax.swing.UIDefaults;
+
+/*
  * @test
  * @bug 6622002
  * @author Alexander Potochkin
```

Comment:
remove @author tag

Reply:
Not specific to JEP 486, this should be done as part of a different issue.

Reply:
agreed

Reply:
there were many tests modified in javax_swing in this PR where the author tag is removed, only this is missed so I pointed it out...